java/ActigraphReader.java

```diff
@@ -147,6 +165,7 @@ else if (tokens[0].trim().equals("Serial Number"))
      **/
     private static void readG3TXV2Epoch(
             InputStream activityReader,
+            String infoTimeShift,
```

Comment:
Aha! So it _is_ a time shift after all, not timezone

Reply:
yeah